libipl/p10/ipl0.C

```diff
@@ -394,12 +394,20 @@ static int ipl_set_ref_clock(void)
 			pdbg_target_index(proc));
 		fapirc = p10_setup_ref_clock(proc);
 		if (fapirc != fapi2::FAPI2_RC_SUCCESS) {
-			ipl_log(IPL_ERROR, "Istep set_ref_clock failed on chip %d, rc=%d\n",
-                                pdbg_target_index(proc), fapirc);
+			ipl_log(IPL_ERROR, "Istep set_ref_clock failed on chip %d, "
+			    "rc=0x%08X \n", pdbg_target_index(proc), fapirc,
+			    fapi2::current_err);
 			rc++;
 		}
 
-		ipl_error_callback((fapirc == fapi2::FAPI2_RC_SUCCESS) ? IPL_ERR_OK : IPL_ERR_HWP);
+		// for plat errors need to callout the proc
```

Comment:
File scope: Callout is based on error type , the statement is good for cfam/scom failures. please remove this statemnet from here.